workers/generic-worker/main.go

```diff
@@ -371,6 +388,9 @@ func HandleCrash(r interface{}) {
 	log.Print(string(debug.Stack()))
 	log.Print(" *********** PANIC occurred! *********** ")
 	log.Printf("%v", r)
+	if err, ok := r.(error); ok && err != nil {
+		errorreport.Send(WorkerRunnerProtocol, err, debugInfo)
+	}
```

Comment:
See above - here I think we can send anything through if we change Send function to receive `interface{}` instead of `error`.